idd/Energy+.idd.in

```diff
@@ -64280,7 +64457,7 @@ WaterHeater:Mixed,
 WaterHeater:Stratified,
        \memo Water heater with stratified, multi-node water tank. May be used to model a tankless
        \memo water heater (small tank volume), a hot water storage tank (zero heater capacity), or
-       \memo a heat pump water heater (see WaterHeater:HeatPump.)
+       \memo a heat pump water heater (see WaterHeater:HeatPump:*.)
```

Comment:
The `WaterHeater:Stratified` tank model can be used with either of the `WaterHeater:HeatPump:*` objects. However, the `WaterHeater:HeatPump:WrappedCondenser` can only be used with the `WaterHeater:Stratified` tank model. I tried to outline the compatibility of the models [here](https://github.com/NREL/EnergyPlus/blob/hpwh_residential/doc/src/docs/EngineeringReference/13f-EncyclopaedicRefs.md#overview)